ext/compressors/snappy/snappy_compress.c

```diff
@@ -26,13 +26,13 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  */
 
+#include <wt_internal.h>
```

Comment:
Why is this moved above the other includes? Generally it's better to include system headers first, so I don't really like this change.